evaluate_iteration_changes.py

```diff
@@ -50,13 +50,16 @@ def collect_iteration_stats(
         updated_sums_by_id = {}
         for sum_id, data in summaries.items():
             if test_set_ids is None or sum_id in test_set_ids:
-                updated_sums_by_id[sum_id] = data["summary"]
-                sums_by_id[sum_id] = data["summary"]
-                sum_ents_by_id[sum_id] = data["labeled_entities"]
-                current_iteration_stats["summary"][sum_id] = {
-                    "summary": data["summary"]
-                }
-                failed_sums_by_id[sum_id] = iteration_idx
+                if data["summary"] == SUMMARY_FAILED_GENERATION:
```

Comment:
Not great that we duplicate logic from load_summaries_from_logs here, but also not worth a bigger refactoring IMO